go/vt/discovery/healthcheck_test.go

```diff
@@ -127,9 +127,9 @@ func TestHealthCheck(t *testing.T) {
 
 	// TabletType changed, should get both old and new event
 	shr = &querypb.StreamHealthResponse{
-		TabletAlias:                         tablet.Alias,
```

Comment:
This is non-standard formatting. Don't you have the pre-commit hooks installed that alert you to this?
All files should be formatted using `gofmt`.

Reply:
while commiting it first asked me to run
goimports -w /home/manan/vitess/go/vt/discovery/healthcheck_test.go
Once I did that and tried to commit it asked me to run
gofmt -s -w /home/manan/vitess/go/vt/discovery/healthcheck_test.go
And then the same thing all over again
So am I supposed to run gofmt or goimports for the files?